network-api/networkapi/management/commands/load_fake_data.py

```diff
@@ -68,6 +69,15 @@ def handle(self, *args, **options):
         social_share_tag, created = Tag.objects.get_or_create(name='social share image')
         images[0].tags.add(social_share_tag)
 
+        # Create one PNI product for every image we have in our media folder
+        product_images = ['babymonitor.jpg', 'drone.jpg', 'nest.jpg',
+                          'teddy.jpg', 'echo.jpg']
+        for image in product_images:
+            create_wagtail_image(
+                f'/app/network-api/media/images/placeholders/products/{image}',
```

Comment:
`/app/...` comes from Docker since that's the standard way to setup a docker project and we follow that rule, thus the hard coded image path.

Reply:
We can use some `os.path` magic in combination with `__file__` here, similar to what we did in the cloudinary migration script

Reply:
Done, good suggestion! We needed this anyway because CI doesnt run seem to have the same paths as our local docker setup (ie. /app/ isnt accessible, or something something that made CI fail)